gap/PolygonalComplexes/properties.gd

```diff
@@ -275,7 +275,13 @@ DeclareAttribute( "BlockType",IsSimplicialSurface);
 #! These two definitions are distinguished by calling them 
 #! <K>EdgeDegreesOfVertices</K> and <K>FaceDegreesOfVertices</K>.
 #! 
-#TODO mention vertexCounter, edgeCounter
+#! A large part of this chapter are the counters. There are six different counter objects:
+#! vertex counter(<Ref Subsect="CounterOfVertices"/>), edge counter(<Ref Subsect="CounterOfEdges"/>),
+#! face counter(<Ref Subsect="CounterOfFaces"/>), butterfly counter(<Ref Subsect="CounterOfButterflies"/>),
+#! umbrella counter(<Ref Subsect="CounterOfUmbrellas"/>) and three face counter(<Ref Subsect="CounterOfThreeFaces"/>).
+#! The counters save information about the degrees. There are different methods to get these information
```

Comment:
Maybe: For a given surface, the counters save information about the vertex, face or edge degrees.